static/elements/utils.js

```diff
@@ -2,18 +2,15 @@
 
 import {html} from 'lit';
 import {unsafeHTML} from 'lit/directives/unsafe-html.js';
-import Autolinker from 'autolinker';
+import {markupAutolinks} from './autolink.js';
```

Comment:
So, can we remove the `autolinker` NPM dependency from package.json now?

Reply:
Yes, we should be able to remove this. Is it worth trying to generate a new `package-lock.json`? Or just simply remove from `package.json`?

Reply:
I'm having a lot of trouble successfully updating package-lock here. This is the correct approach, but I'll handle this in a separate PR.